shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -71,8 +70,6 @@ public int run() throws IOException {
     operations.add("PathsUnmounted");
 
     mPrintStream.println("Alluxio logical operations: ");
-    metricsMap.put("FilesPinned", metricsMap.get("master.FilesPinned"));
-    metricsMap.remove("master.FilesPinned");
```

Comment:
can we improve this code to handle the possibility that the value for one of the metrics could be `null`? Instead of throwing NPE, we could ignore the metric, or print "null" or "unknown" as the value for just that metric